### PR TITLE
Const paramters

### PR DESCRIPTION
**Pull request by lilAmper**
Const added Request, RequestJSON, JsonStringify..etc functions.

### CODE REVIEW

File: requests.inc [hunk 1]
```diff
@@ -80,7 +80,7 @@ enum JSON_NODE {
 native JsonParse(const string[], &Node:output);
 
 // JsonStringify encodes a JSON node into `buf`.
-native JsonStringify(Node:node, buf[], len = sizeof(buf));
+native JsonStringify(Node:node, const buf[], len = sizeof(buf));
```
Comment (Alasnkz) [inline requests.inc:83]: Shouldn't this be left, being as buf is modified?

Comment (lilAmper) [reply requests.inc:83]: Doesn't it make more sense for the parameter to be constant? after all, why edit it after this function?

Comment (Sreyas-Sreelal) [reply requests.inc:83]: The native converts JSON into string representation and stores the result in buf therefore it shouldn't be constant.

Comment (lilAmper) [reply requests.inc:83]: Ah, okay. Thank you both!

**Pull request closed without merging**